app/controllers/providers/proceedings_types_controller.rb

```diff
@@ -32,6 +35,20 @@ def update
 
     private
 
+    def run_transaction
+      proceeding_types_service.add(proceeding_type_id: form_params, scope_limitation: :substantive)
```

Comment:
Hmm, looks like the controller stills only adds 1 proceeding type rather than multiple (but maybe that's still the intention for now and that this be developed further in a future ticket). Have sent you DM to check if this is so.